spec/services/hyrax/collections/nested_collection_query_service_spec.rb

```diff
@@ -309,8 +268,9 @@
       let(:parent) { coll_e }
       let(:child) { another }
 
-      it 'returns false' do
-        expect(subject).to eq false
+      # Limit no longer applies so it will always return true
+      it 'returns true' do
+        expect(subject).to eq true
```

Comment:
We could use solr's `maxDepth` parameter to the graph query parser, but that feels like more work than it is worth.

Reply:
that makes sense to me. 

If you exceed it there should just be a message that says "you have delved too deep. the Balrog comes"